packages/components/src/tools-panel/styles.ts

```diff
@@ -107,6 +108,15 @@ export const ToolsPanelItem = css`
 			margin-bottom: 0;
 		}
 	}
+
+	/* Standardize InputControl labels with others inside ToolsPanel. */
```

Comment:
We should add a comment here about this being a temporary fix — with the proper fix being the normalization of the different control components